### PR TITLE
OCPRHV-374 add a note pointing people to kcs letting people know abou… [enterprise-4.5]

### PR DESCRIPTION
**Pull request by rolfedh**
…t broken ocp 4.5 on rhv 4.4

https://issues.redhat.com/browse/OCPRHV-374

### CODE REVIEW

File: installing/installing_rhv/installing-rhv-customizations.adoc [hunk 1]
```diff
@@ -5,6 +5,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+[WARNING]
+====
+Due to a known issue with installing {product-title} versions 4.4 and 4.5 on {rh-virtualization-first} 4.4., you must customize `install-config.yaml` as described in link:https://access.redhat.com/solutions/5374251[Openshift IPI installation on RHV-4.x failed with "Error: timeout while waiting for state to become 'up' (last state: 'down', timeout: 10m0s)"].
```
Comment (peterclauterbach) [inline installing/installing_rhv/installing-rhv-customizations.adoc:10]: specifically RHV 4.4.1

Comment (peterclauterbach) [inline installing/installing_rhv/installing-rhv-customizations.adoc:10]: This defect is fixed in RHV 4.4.2

File: installing/installing_rhv/installing-rhv-default.adoc [hunk 2]
```diff
@@ -5,6 +5,15 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+[WARNING]
+====
+Due to a known issue, this procedure does not work with {product-title} versions 4.4 and 4.5 on {rh-virtualization-first} 4.4.
```
Comment (peterclauterbach) [inline installing/installing_rhv/installing-rhv-default.adoc:10]: the default install procedure

**Review by peterclauterbach — APPROVED** [review]

File: installing/installing_rhv/installing-rhv-default.adoc (hunk 2 above)
Comment (ahardin-rh) [reply installing/installing_rhv/installing-rhv-default.adoc:10]: Yeah, I would say "the default installation procedure"

File: installing/installing_rhv/installing-rhv-customizations.adoc (hunk 1 above)
Comment (vikram-redhat) [inline installing/installing_rhv/installing-rhv-customizations.adoc:10]: @rolfedh minor typo (`4.4.,`)

**Comment by ahardin-rh** [comment]
Confirmed that this issue is fixed in 4.6 and, therefore, this should only be cherry-picked to 4.5.

**Pull request merged**